static/js/formatters-internal.js

```diff
@@ -283,22 +283,28 @@ export function joinList(list, separator) {
 }
 
 /**
- * Given an image object with a url, changes the url to use dynamic thumbnailer and https.
+ * Given an image object with a url, changes the url to be dynamic and use https.
  *
- * Note: A dynamic thumbnailer url generated with atLeastAsLarge = true returns an image that is
- * at least as large in one dimension of the desired size. In other words, the returned image will
- * be at least as large, and as close as possible to, the largest image that is contained within a
- * box of the desired size dimensions.
+ * If both dimensions are provided in desiredSize (e.g. "200x100"):
+ *   - if atLeastAsLarge = true, returns an image that covers the desired space while preserving
+ *     the original image ratio (e.g. if original image is 100x100, returns an image of 200x200).
+ *     Note that if we can't get the original image ratio from the image object, the behavior would
+ *     be similar to when atLeastAsLarge = false.
+ *   - if atLeastAsLarge = false, returns an image that is as large in at least one dimension while
+ *     preserving the original image ratio (e.g. if original image is 100x100, returns an image of
+ *     100x100).
  *
- * If atLeastAsLarge = false, the dynamic thumbnailer url will give the largest image that is
- * smaller than the desired size in both dimensions.
+ * If only one dimension is provided in desiredSize (e.g. "200x", which is also the default):
+ *   - returns an image that matches this dimension while preserving ratio of the original image
+ *     (e.g. if original image is 100x100, returned image would be 200x200).
+ *
+ * If "" is provided as the desiredSize, returns an image in the original size.
```

Comment:
can this be `If "", "x", "0x0", or a string with unexpected format is provided as the desiredSize ....`? We may want to call out that 0 values are disregarded